UPDATE BE

tiennguyen-developer opened the pull request:
1. Create Shoppify_service
2. Create Shoppify_data for MySql
3. Build Flyway connect MySQL

Commit:
UPDATE BE
1. Create Shoppify_service
2. Create Shoppify_data for MySql
3. Build Flyway connect MySQL

sn0509 commented on src/main/java/com/shoppify/entity/Role.java:
```diff
@@ -0,0 +1,33 @@
+   @Column(name = "DESCRIPTION")
+   private String description;
+
+   @ManyToOne
```
Change to Many-to-Many

sonnguyen-2000 commented on src/main/java/com/shoppify/entity/User.java:
```diff
@@ -0,0 +1,60 @@
+   @Column(name="IS_DELETED")
+   private boolean isDeleted;
+
+   @Column(name = "FULL_NAME")
```
These attributes should be separated into a new table with the name user_profile

sonnguyen-2000 commented on src/main/java/com/shoppify/entity/Role.java:
```diff
@@ -0,0 +1,33 @@
+   @Column(name = "NAME")
+   private String name;
+
+   @Column(name = "DESCRIPTION")
```
remove this addtribute

sonnguyen-2000 commented on src/main/java/com/shoppify/entity/Address.java:
add new attribute "fullname" here

Commit:
UPDATE BE v2
1. Create Shoppify_service
2. Create Shoppify_data for MySql
3. Build Flyway connect MySQL

sonnguyen-2000 commented on src/main/java/com/shoppify/entity/Role.java:
```diff
@@ -0,0 +1,33 @@
+   @Column(name = "DESCRIPTION")
+   private String description;
+
+   @ManyToOne
```
remove @OnetoOne and change list<User> to Set<User> to ensure avoidance of duplication

tiennguyen-developer replied:
Một User sẽ có nhiều địa chỉ em
